packages/backend/src/api/routers/ForcedTradeOfferRouter.ts

```diff
@@ -0,0 +1,72 @@
+import { AssetId, StarkKey } from '@explorer/types'
+import Router from '@koa/router'
+import bodyParser from 'koa-bodyparser'
+import { string, z } from 'zod'
+
+import { ForcedTradeOfferController } from '../controllers/ForcedTradeOfferController'
+import {
+  stringAs,
+  stringAsBigInt,
+  stringAsInt,
+  withTypedContext,
+} from './types'
+import { applyControllerResult } from './utils'
+
+export function createForcedTradeOfferRouter(
+  offerController: ForcedTradeOfferController
+) {
+  const router = new Router()
+
+  router.post(
+    '/offer',
+    bodyParser(),
+    withTypedContext(
+      z.object({
+        request: z.object({
+          body: z.object({
+            starkKeyA: stringAs(StarkKey),
+            positionIdA: stringAsBigInt(),
+            syntheticAssetId: stringAs(AssetId),
+            amountCollateral: stringAsBigInt(),
+            amountSynthetic: stringAsBigInt(),
+            aIsBuyingSynthetic: z.boolean(),
+          }),
+        }),
+      }),
+      async (ctx) => {
+        const result = await offerController.postOffer(ctx.request.body)
+        applyControllerResult(ctx, result)
+      }
+    )
+  )
+
+  router.post(
+    '/offer/:initialOfferId/accept',
```

Comment:
`router.put('offer/:initialOfferId')`, we use PUT, because we modify the resource at this url